test/mocha/30-backwards-compatibility.js

```diff
@@ -0,0 +1,536 @@
+import {createRequire} from 'node:module';
+import {httpClient} from '@digitalbazaar/http-client';
+import {klona} from 'klona';
+import {mockData} from './mock.data.js';
+
+const require = createRequire(import.meta.url);
+
+const {baseUrl} = mockData;
+const serviceType = 'vc-issuer';
+
+// NOTE: using embedded context in mockCredential:
+// https://www.w3.org/2018/credentials/examples/v1
+const mockCredential = require('./mock-credential.json');
+
+describe('issue APIs - Reference id `assertionMethod:ed25519` backwards ' +
+  'compatibility test', () => {
+  const zcaps = {};
+  describe('Ed25519Signature2020', function() {
+    let capabilityAgent;
+    let noStatusListIssuerId;
+    let noStatusListIssuerRootZcap;
+    let rl2020IssuerId;
+    let rl2020RootZcap;
+    let sl2021RevocationIssuerId;
+    let sl2021RevocationRootZcap;
+    let sl2021SuspensionIssuerId;
+    let sl2021SuspensionRootZcap;
+    let oauth2IssuerConfig;
+    beforeEach(async () => {
+      const suiteName = 'Ed25519Signature2020';
+      const secret = '53ad64ce-8e1d-11ec-bb12-10bf48838a41';
+      const handle = 'test';
+      capabilityAgent = await CapabilityAgent.fromSecret({secret, handle});
+
+      // create keystore for capability agent
+      const keystoreAgent = await helpers.createKeystoreAgent(
+        {capabilityAgent});
+      // generate key for signing VCs (make it a did:key DID for simplicity)
+      const assertionMethodKey = await keystoreAgent.generateKey({
+        type: 'asymmetric',
+        publicAliasTemplate: 'did:key:{publicKeyMultibase}#' +
+              '{publicKeyMultibase}'
+      });
+
+      // create EDV for storage (creating hmac and kak in the process)
+      const {
+        edvConfig,
+        hmac,
+        keyAgreementKey
+      } = await helpers.createEdv({capabilityAgent, keystoreAgent});
+
+      // get service agent to delegate to
+      const serviceAgentUrl =
+          `${baseUrl}/service-agents/${encodeURIComponent(serviceType)}`;
+      const {data: serviceAgent} = await httpClient.get(
+        serviceAgentUrl, {agent});
+
+      // delegate edv, hmac, and key agreement key zcaps to service agent
+      const {id: edvId} = edvConfig;
+      zcaps.edv = await helpers.delegate({
+        controller: serviceAgent.id,
+        delegator: capabilityAgent,
+        invocationTarget: edvId
+      });
+      const {keystoreId} = keystoreAgent;
+      zcaps.hmac = await helpers.delegate({
+        capability: `urn:zcap:root:${encodeURIComponent(keystoreId)}`,
+        controller: serviceAgent.id,
+        invocationTarget: hmac.id,
+        delegator: capabilityAgent
+      });
+      zcaps.keyAgreementKey = await helpers.delegate({
+        capability: `urn:zcap:root:${encodeURIComponent(keystoreId)}`,
+        controller: serviceAgent.id,
+        invocationTarget: keyAgreementKey.kmsId,
+        delegator: capabilityAgent
+      });
+      zcaps[`assertionMethod:${assertionMethodKey.algorithm}`] = await helpers
```

Comment:
Shouldn't this say `assertionMethod:ed25519` for a backwards compatibility test?

Reply:
Oh, hmm, right we aren't accepting this any more, I'll look more closely to see how we change it in the backend.